netlify/next-runtime
# fix: correctly match params in edge runtime

### Summary

Next.js was not populating the params from dynamic edge runtime pages. This PR fixes that by populating the `x-matched-path` header, which along with setting minimal mode, makes Next.js parse the params as if we're on a certain triangle-shaped host. This includes correctly populating the params.

Fixes #1890

### Test plan

1. Visit `/edge/123?hello=world` in the default demo
2. Check that query is `{ id: '123', hello: 'world' }` and params is `{ id: '123' }`

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal
![pão em forma de capivara](https://user-images.githubusercontent.com/213306/213765573-338666b3-76a4-43a0-beb5-779eb6ecfea0.png)

### Standard checks:




---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
